src/core/StelToneReproducer.cpp

```diff
@@ -89,56 +89,5 @@ void StelToneReproducer::setWorldAdaptationLuminance(float _Lwa)
 	alphaWaOverAlphaDa = alphaWa/alphaDa;
 	term2 = (stelpow10f((betaWa-betaDa)/alphaDa) / (M_PIf*0.0001f));
 	lnTerm2 = std::log(term2);
-	term2TimesOneOverMaxdLpOneOverGamma = std::pow(term2*oneOverMaxdL, oneOverGamma);
-}
-
```

Comment:
Just a few days ago we had a request to add sky brightness etc. data to the mouse position coordinate plugin, which I basically agree to add when the numerical data make sense. This should deliver Y [cd/m^2] (coming from Schaefer's sky model?) and could of course deliver also Yxy or CIE RGB reference data if this code is still available. (There may be some use in RemoteControl connected apps to be yet developed.) I think it is simpler to call such a function for it than to get samples from the GPU (?) So please, instead of deleting, double check to see if this is the standardized CIE formulation, and update documentation.

Reply:
I'm not sure what you mean. This hunk just replaces one cached term (that includes `...^(1/γ)`) with another (that doesn't include it).

Reply:
My concern is that by removing the whole conversion function (xyYToRGB()) may remove functionality that we were asked to make more accessible just a few days ago. I recommend double-check to ensure these are current CIE RGB values, document more clearly, and indeed implement that lookup/feedback in the mouse coordinate plugin (like a virtual sky brightness sensor).

Reply:
@10110111 see issue #1404 for details

Reply:
@gzotti 
As I see in #1404, it asks for _luminance_. Luminance is the _input_ for `xyYToRGB()`. Moreover, even if someone asked us for RGB values (be it sRGB, CIE RGB or whatever else kind of RGB), Tone Reproducer is a bad object to ask for them.

Why? Because it's not meant to provide anything objective at all. It's supposed to take objective data (CIE xyY color coordinates) and try to simulate _perception_ of these colors, adapting them to the user's display device and the assumed user's color adaptation. To this end, it uses XYZ-to-sRGB conversion matrix in one case, does blue-shifting in other, or simply substitutes a constant blue in the rest of cases. Plus gamma correction.

This means that, if we wanted to use `xyYToRGB()` to satisfy #1404, we'd better stop and just return the Y parameter as the luminance requested. If we had another feature request that asks for chromaticity, we should take the x,y,Y parameters and do the conversion to whatever color space desired. But in no case should we even _touch_ Tone Reproducer for these purposes. It could just as well not exist at all, without any effect on these features.

Reply:
Yes, this one only asks for luminance. But we could provide other data as well when we already have them. SB else (not on github) asked me for a "sky color query".  This may simply sample Stellarium's sky, but I am not yet sure what best to return (I have currently no time to go into this). CIE Yxy, CIE XYZ, CIE RGB, sRGB, ...?

Reply:
If you want true absolute colors, the best is either xyY or XYZ. Any RGB variant, I suppose, is not too useful: sRGB may in general become negative for very saturated colors, and also has its gamma, CIE RGB doesn't seem to be used anywhere. XYZ is always non-negative and widely used. xyY is when you want to separate chromaticity from luminance in the XYZ coordinates.

Anyway, removal of Tone Reproducer's `xyYToRGB` doesn't affect this in the slightest.